static/sass/partials/_header.scss

```diff
@@ -1,19 +1,129 @@
 /* SASS file for header. */
 header {
+	/*Sitename and menu.*/
+	.sitename-wrapper {
+		.desktop-menu {
+			display: none;
+		}
+		border-bottom: 0.2px solid $border-btn-color;
+			.text-logo {
+				text-transform: uppercase;
+				text-align: left;
+				margin: 15px 0 22px 0;
+				font-family: $text-font;
+				padding-top: 4px;
+				margin-left: 5px;
+					a {
+						font-size: 28px;
+						font-weight: bold;
+						color: $primary-color;
+						text-decoration: none;
+					}
+				.text-logo-second {
+					color: $secondary-color;padding: 0;
+				}
+			}
+		span {
+			margin-top: 17px;
+			padding-left: 27px;
+		}
+		.menu-wrapper {
+			text-transform: uppercase;
+	    height: 100%;
+	    width: 0;
+	    position: fixed;
```

Comment:
I will check with you here.

Reply:
This is needed for hamburger menu display.